### PR TITLE
Add okteto development flow

### PR DESCRIPTION
**Pull request by pchico83**
This PR implements the deployment of the `gglsbl-rest` service on Okteto Cloud with one click, and add supports to develop the code directly on Okteto Cloud to reduce local setup.

Happy to talk more about this use case.
You can find me as **pablo** on the #okteto channel of the Kubernetes community Slack:
https://slack.okteto.com

### CODE REVIEW

File: .stignore [hunk 1]
```diff
@@ -0,0 +1,101 @@
+.git
```
Comment (asieira) [inline .stignore:1]: If I understand correctly this is a local configuration file of a tool you use on your development machine. Please remove it from the PR.

File: Dockerfile [hunk 2]
```diff
@@ -9,16 +11,15 @@ RUN apk update && \
 ## Populate app directory
 WORKDIR /home/gglsbl
 ENV GSB_DB_DIR /home/gglsbl/db
-COPY ["requirements.txt", "*.py", "logging.conf", "./"]
+COPY ["requirements.txt", "./"]
 ENV LOGGING_CONFIG /home/gglsbl/logging.conf
 
 # Install Python packages, cleanup and set permissions
-RUN pip3 install --upgrade pip setuptools && \
+RUN --mount=type=cache,target=/root/.cache \
```
Comment (asieira) [inline Dockerfile:18]: These extra parameters break building this on plain old Docker, please make sure that whatever changes you make to the Dockerfile keep backwards compability with `docker build`.

**Review by asieira — CHANGES_REQUESTED** [review]
Thank you for this contribution! I have no experience with Okteto myself, but I look forward to adding support for it to the project.

File: Dockerfile (hunk 2 above)
Comment (pchico83) [reply Dockerfile:18]: Makes sense, I will update the PR

File: .stignore (hunk 1 above)
Comment (pchico83) [reply .stignore:1]: This is a configuration file for Okteto. Okteto works by synchronizing local changes to a development container. This file indicates which files should not be synched.

**Comment by pchico83** [comment]
@asieira updated.
Okteo gives you two main use cases:
- Deploy your app in a single click (we have seen two users trying to deploy this repo)
- Develop your app in a remote development container, instead of installing/running anything locally.

**Review by asieira — APPROVED** [review]

**Pull request merged**